wallet/how-to/use-non-evm-networks/starknet-snap/index.md

```diff
@@ -0,0 +1,174 @@
+---
+sidebar_position: 1
+sidebar_label: Starknet
+description: Learn about how Starknet Snap connects to MetaMask using `get-starknet`.
+tags:
+  - Snaps
+  - non-EVM
+---
+
+# Build with Starknet
+
+Starknet is a Layer 2 network built on Ethereum. It processes transactions off the main Ethereum chain to increase speed and reduce costs, while still benefiting from Ethereum's security.
+
+One major aspect of Starknet development is integrating with user wallets. There are two primary approaches to achieve this integration:
+
+- [Using direct Snap invocation with `wallet_invokeSnap](#use-wallet_invokesnap-directly)
+- [Using the `get-starknet` integration](#use-the-get-starknet-integration)
+
+Both methods enable developers to connect Starknet dapps with wallet solutions. 
+The choice between them depends on the specific needs of the project, the desired level of control, and familiarity with Starknet and Snap technologies.
```

Comment:
Shall we make clear the benefits of using get-starknet ? 

From get-starknet stand point they are **the wallet interoperability layer** of the starknet ecosystem. This means that a starknet dApp developer choosing get-starknet to develop will automatically be compatible with several wallet at the same time (including metamask, through the starknet-snap).

Reply:
I think that is mentioned later on